include/gfxbase.h

```diff
@@ -298,16 +298,12 @@ class GFXBase : public Adafruit_GFX
     {
         if (isValidPixel(x, y))
             leds[XY(x, y)] = color;
-        else
-            debugE("Invalid drawPixel request: x=%d, y=%d, NUM_LEDS=%d", x, y, NUM_LEDS);
     }
 
     void drawPixel(int16_t x, int16_t y, uint16_t color) override
     {
         if (isValidPixel(x, y))
             leds[XY(x, y)] = from16Bit(color);
-        else
```

Comment:
Actually, they're not always. The error messages were removed because a dependency that can be used to move text across the panel doesn't do boundary checking. With the validity check already ignoring out of bounds pixels, it seemed unnecessary to also create a lot of debugging noise.